ingest/ingest_integration_test.go

```diff
@@ -181,7 +149,7 @@ var _ = Describe("Ingestion", func() {
 			Expect(err).To(Succeed())
 			Expect(resp2).NotTo(BeNil())
 			id2 := resp2.Info.Changes[0].Id
-			Expect(id2).To(Equal(id))
+			Expect(id2).NotTo(Equal(id))
```

Comment:
Wait, so you ingest the exact same node twice, and receive different ids?

Reply:
yes the behavior changed: before, when you created the same node twice, it did not create a new one but was just updating the old one.
Now it is the contrary: it delete the old one and create a new one.
Same for Resources which are not DT: the behavior changes too.